docs/source/conf.py

```diff
@@ -63,3 +63,7 @@
 html_static_path = []
 
 master_doc = "index"
+
+# user starts in light mode
+
+default_dark_mode = False
```

Comment:
Yes, this is what we want, light stays as default, as that is the case without this plugin.

Reply:
> Yes, this is what we want, light stays as default, as that is the case without this plugin.

Yep ✨